### PR TITLE
Data release 8/24/2022

### PR DESCRIPTION
**Pull request by beets**

### CODE REVIEW

File: internal/server/v0/triple/golden/get_triples/place_type.json [hunk 1]
```diff
@@ -92,7 +92,7 @@
       "objectTypes": [
         "Class"
       ],
-      "provenanceId": "dc/5n63hr1"
+      "provenanceId": "dc/sgvysq3"
```
Comment (beets) [inline internal/server/v0/triple/golden/get_triples/place_type.json:95]: Is this provenance diff expected? I believe it's from this import:
https://github.com/datacommonsorg/data/pull/736/files#diff-5ebef208962874063879510d265089ec2da98f628b22151afbf29ef23df7a4c3

which defines country nodes to use the resolver.

/cc @pulkit-s

File: internal/server/v0/internalbio/golden/get_bio_page_data/yeast.json [hunk 2]
```diff
@@ -27,9 +27,6 @@
                           "property": "value",
                           "direction": "DIRECTION_OUT",
                           "nodes": [
-                            {
-                              "value": "0.35"
-                            },
```
Comment (beets) [inline internal/server/v0/internalbio/golden/get_bio_page_data/yeast.json:32]: @shifucun is this deletion expected?

**Review by shifucun — APPROVED** [review]

File: internal/server/v0/internalbio/golden/get_bio_page_data/yeast.json (hunk 2 above)
Comment (shifucun) [reply internal/server/v0/internalbio/golden/get_bio_page_data/yeast.json:32]: Yes, this is a bug that I fixed in https://github.com/datacommonsorg/mixer/commit/830a4f19f7ddb3c530128432c00ac23b7e703c29. If you sync, the diff should go away.

File: internal/server/v0/triple/golden/get_triples/place_type.json (hunk 1 above)
Comment (shifucun) [reply internal/server/v0/triple/golden/get_triples/place_type.json:95]: The provenance is non-deterministic. I think we need a proper fix for this longer term.

File: internal/server/v0/internalbio/golden/get_bio_page_data/yeast.json (hunk 2 above)
Comment (beets) [reply internal/server/v0/internalbio/golden/get_bio_page_data/yeast.json]: Thanks - just saw your fix too.

**Pull request merged**